### DISCUSSION TITLE
Just added documentation on how to add a new symbol

### DISCUSSION THREAD

**Discussion by toolness** [mysticsymbolic/mysticsymbolic.github.io#168] [Show and tell]
Hey everyone, I just added some documentation on how to add new symbols in #166.  You can see the documentation [here](https://github.com/mysticsymbolic/mysticsymbolic.github.io/blob/main/assets/symbols/_instructions.md) (it's also linked to from the README).  Let me know if you have any thoughts, or if anything is unclear!

/cc  @ninapaley

**Comment by ninapaley** [comment]
Thanks @toolness ! 

There were two additional lines in the template.toml file I have locally; I just took the liberty of adding them to https://github.com/mysticsymbolic/mysticsymbolic.github.io/blob/main/assets/symbols/_template.toml. Please check that I didn't screw anything up. Thanks!
